bin.js

```diff
@@ -50,9 +51,16 @@ if (argv.help || !sourceArg || !destArg) {
 const source = path.resolve(process.cwd(), sourceArg)
 const dest = path.resolve(process.cwd(), destArg)
 
-zipIt.zipFunctions(source, dest, { skipGo: !argv['zip-go'] })
-.then(console.log)
-.catch(err => {
+/* Recursively loop through functions + function folders and install deps */
+installFunctionDeps(source).then(() => {
+  /* Then zip em up */
+  zipIt.zipFunctions(source, dest, { skipGo: !argv['zip-go'] })
+  .then(console.log)
```

Comment:
Duplicate promise chain path? Maybe just return the promise from the arrow function and handle the chain in a non-nested manner?

Reply:
Unless there is a reason to do it this way and I'm not seeing it.

Reply:
Flattened it!